AmethystAPI/src/minecraft/src-client/common/client/gui/ScreenView.h

```diff
@@ -1,11 +1,15 @@
 #pragma once
 #include "minecraft/src-client/common/client/renderer/screen/MinecraftUIRenderContext.h"
+#include "minecraft/src-client/common/client/gui/gui/VisualTree.h"
 
 // Not sure if this is correct..
 using UIRenderContext = MinecraftUIRenderContext;
 
 class ScreenView {
 public:
+
+    BUILD_ACCESS(this, class VisualTree*, VisualTree, 0x48);
```

Comment:
This should be replaced with an actual field in the struct, see: https://github.com/FrederoxDev/Amethyst/pull/39#issuecomment-1895228515